Library/Formula/cgdb.rb

```diff
@@ -18,9 +18,14 @@ class Cgdb < Formula
     depends_on "automake" => :build
   end
 
+  if OS.linux?
+    depends_on "flex" => :build
+  end
   depends_on "help2man" => :build
   depends_on "readline"
 
+  patch :DATA
```

Comment:
I haven't submitted it upstream. I tested cgdb from master and it doesn't have this issue. I believe it is only relevant to the released tarball.